vsphere/datadog_checks/vsphere/utils.py

```diff
@@ -99,35 +99,41 @@ def get_parent_tags_recursively(mor, infrastructure_data, config):
           HOST2
 
     """
-    mor_props = infrastructure_data[mor]
-    parent = mor_props.get('parent')
-    if parent:
-        tags = []
-        parent_props = infrastructure_data.get(parent, {})
-        parent_name = to_string(parent_props.get('name', 'unknown'))
-        if isinstance(parent, vim.HostSystem):
-            tags.append('vsphere_host:{}'.format(parent_name))
-        elif isinstance(parent, vim.Folder):
-            if isinstance(parent, vim.StoragePod):
-                tags.append('vsphere_datastore_cluster:{}'.format(parent_name))
-                # Legacy mode: keep it as "folder"
-                if config.include_datastore_cluster_folder_tag:
-                    tags.append('vsphere_folder:{}'.format(parent_name))
-            else:
+    tags = []
+    parent_props = infrastructure_data.get(mor, {})
```

Comment:
Maybe a naming issue... but isn't the parent the entity bound to the `parent` property in `InfrastructureData` ? What you name `parent_props` here looks like the properties of the `mor` instance we are currently considering. 
Should we rename `parent_props` to just `props`?

Reply:
Yes, you are right. I didn’t renamed it during the refactoring, thx for pointing out that.
Fixed